restful_api/myapi/manage.py

```diff
@@ -14,11 +14,11 @@ def init():
     from myapi.extensions import db
     from myapi.models import User
 
-    click.echo("create user")
-    user = User(username="testuser", email="testuser@mail.com", password="testpassword", active=True)
-    db.session.add(user)
-    db.session.commit()
-    click.echo("created user admin")
+    # click.echo("create user")
+    # user = User(username="testuser", email="testuser@mail.com", password="testpassword", active=True)
+    # db.session.add(user)
+    # db.session.commit()
+    # click.echo("created user admin")
```

Comment:
I forgot to uncomment this. I originally commented this out as `manage.py` isn't v useful in development phase due to isolation from pytest.